evaluation/auto_evaluation/script_based/eval_benchmark.py

```diff
@@ -0,0 +1,355 @@
+import os
+import csv
+from typing import List, Dict, Any
+from tqdm import tqdm
+import requests
+import time
+import json
+from vertexai import init as vertexai_init
+from vertexai.generative_models import GenerativeModel, HarmCategory, HarmBlockThreshold
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
+load_dotenv()
+
+# Use environment variables
+BASE_URL = os.getenv('ENDPOINT1')
+BASE_URL_AGENT_RERANKER = os.getenv('ENDPOINT2')
+INPUT_FILE = os.getenv('input_file_path', 'data.csv')  # Use 'data.csv' as default if not specified
+
+# Configuration constants
+USE_CLI_INPUT = True # Set to False to use the constants below instead of CLI input
+
+# If USE_CLI_INPUT is False, the following constants will be used
+SELECTED_RETRIEVERS = ["all"]  # or use ["all"] for all retrievers
+ITERATIONS = 1
+
+# Default LLM model configuration
+MODEL_NAME = "gemini-1.5-pro"
+generation_config = {
+    "temperature": 0,
+    "response_mime_type": "text/plain",
+}
+
+safety_settings = {
```

Comment:
What is the default value for this i am wondering. also for production we need to think whether to enable some blocking of harmful outputs.

Reply:
> What is the default value for this i am wondering. also for production we need to think whether to enable some blocking of harmful outputs.

its default, cause there was issue with the safety settings, sometimes it would block certain questions without having any harmful content which hinders the evaluation script.

Reply:
That's fine, but I was curious what is the default `safety` level. I get that we have to disable it for evaluation, but for production we might need it back since we didn't set it for our backend code.